xblock/fields.py

```diff
@@ -829,6 +838,11 @@ def to_string(self, value):
         """String gets serialized and deserialized without quote marks."""
         return self.to_json(value)
 
+    @property
+    def none_to_xml(self):
+        """Returns True to use a XML node for the field and represent None as an attribute."""
+        return True
```

Comment:
Using a `@property` decorator in this class makes the attribute immutable for this class. So no code can change a String field to *not* send Nones to XML. It's not that important - just an extra safeguard.